internal/infrastructure/storage/db/test/withdrawal_repository_impl_test.go

```diff
@@ -0,0 +1,108 @@
+package db
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tdex-network/tdex-daemon/internal/core/ports"
+	"github.com/tdex-network/tdex-daemon/internal/infrastructure/storage/db/inmemory"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/tdex-network/tdex-daemon/internal/core/domain"
+
+	dbbadger "github.com/tdex-network/tdex-daemon/internal/infrastructure/storage/db/badger"
+)
+
+func TestWithdrawalRepositoryImplementations(t *testing.T) {
+	repositories := createWithdrawalRepositories(t)
+
+	for i := range repositories {
+		repo := repositories[i]
+
+		t.Run(repo.Name, func(t *testing.T) {
+			t.Parallel()
+
+			t.Run("testAddAndListWithdrawals", func(t *testing.T) {
+				t.Parallel()
+				testAddAndListWithdrawals(t, repo)
+			})
+		})
+	}
+}
+
+func testAddAndListWithdrawals(t *testing.T, repo withdrawalRepository) {
+
```

Comment:
Same here, and please remove blank line.

Reply:
Done